src/renderer/services/neo.js

```diff
@@ -326,17 +327,33 @@ export default {
       try {
         const inMemory = _.get(store.state.transactionDetails, hash);
         if (inMemory) {
-          inMemory.currentBlockHeight = network.getSelectedNetwork().bestBlock.index;
-          inMemory.confirmations = inMemory.currentBlockHeight - inMemory.block;
+          if (network.getSelectedNetwork().bestBlock) {
+            inMemory.currentBlockHeight = network.getSelectedNetwork().bestBlock.index;
```

Comment:
I'm assuming this is to set a height on some kind of element. Is it not possible to style using flex box in this instance?

Reply:
It is not related to a height of any element, it is the block height from the blockchain.